src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -45,6 +45,27 @@ namespace application_manager {
 
 namespace commands {
 
+namespace {
+/**
+ * @brief Functor for build info string
+ */
+struct InfoAppender {
+  explicit InfoAppender(std::string& info) : info_(info) {}
+
+  void operator()(const RPCParams::value_type& parameter) {
+    if (info_.empty()) {
+      info_ = "\'" + parameter + "\'";
+      return;
+    }
+
+    info_ = info_ + ", \'" + parameter + "\'";
+  }
+
+ private:
+  std::string& info_;
+};
+}  // namespace
+
```

Comment:
@ZhdanovP Think about lambda instead of functor...

Reply:
@AByzhynar Hello, we use InfoAppender for different info : `disallowed_by_user_info` and `disallowed_by_policy_info` and of course we can use lambda here, but I am not sure that it will be more readable  and useful.